tests/sys_crypto/tests-crypto-modes-cbc-padding.c

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright (C) 2017 Mathias Tausig
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+#include <stdio.h>
+#include <string.h>
+
+#include "embUnit.h"
+#include "crypto/padding.h"
+#include "tests-crypto.h"
+
+static uint8_t TEST_DATA[] = {
+    0x2b, 0x7e, 0x15, 0x16, 0x28, 0xae, 0xd2, 0xa6,
+};
+
+static uint8_t TEST_1_BLOCKSIZE = 16;
+static uint8_t TEST_1_PADDED_DATA[] = {
```

Comment:
Are these public test vectors? If so, please mention the source (NIST xyz). If not, did you check for existing test vectors?

Reply:
There are no specific test vectors for padded AES ciphers. I just used the same vectors that were used in `tests/sys_crypto/tests-crypto-modes-cbc.c`, which are NIST vectors, according to the documentation.